platform/ios/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 * Fixed an issue where `-[MGLMapView metersPerPixelAtLatitude:]` was removed, but not marked as unavailable. ([#11765](https://github.com/mapbox/mapbox-gl-native/pull/11765))
 * Fixed an issue where selecting an onscreen annotation could move the map unintentionally. ([#11731](https://github.com/mapbox/mapbox-gl-native/pull/11731))
+* Reduced startup time by causing MGLMapboxEvents to be setup asynchronously. (https://github.com/mapbox/mapbox-gl-native/pull/11784)
```

Comment:
This changelog’s audience is developers who install the SDK. MGLMapboxEvents isn’t a public class, so let’s leave it at:

> Improved application launch performance.